patches/v8/0005-Revert-bazel-Add-hide-symbols-from-release-fast-buil.patch

```diff
@@ -17,29 +17,23 @@ dynamic linking of unit tests in fastbuild mode. A subsequent reland of
 the change uses the flags only in opt mode, which should work better.
 
 This patch can be removed when updating to V8 10.9.
```

Comment:
Can this patch be removed now?

Reply:
Possibly. The project builds and tests locally with `bazel test -c fastbuild ...` with this patch removed from the sequence. Let me know if there is any additional test we should do here, thanks!

Reply:
Nope. This was purely a build breakage in `fastbuild` mode and was subsequently fixed upstream. Sounds like it can be removed.